### ISSUE TITLE
Bug: Debugger doesn't work with ALLSELECTED properly.

### ISSUE THREAD

**Issue by edhans** [TabularEditor/TabularEditor3#669]
Here is my measure. (this is in a standard Contoso file)
```
% of Total Sales AllSelected =
VAR varSales = [Total Net Sales]
VAR varAllSales =
    CALCULATE(
        [Total Net Sales],
        ALLSELECTED('Products')
    )
VAR Result =
    DIVIDE(
        varSales,
        varAllSales
    )
RETURN
    Result
```

It allows me to generate this visual where my total remains 100% no matter what I select. Pretty basic.
<img width="537" alt="image" src="https://user-images.githubusercontent.com/45298358/191791126-a7053f39-764c-4921-90c7-7767db02c554.png">

When I debug this though, either the debugger is wrong, or I am not understanding it. I ran the query and copied it to TE/3.
```
// DAX Query
DEFINE
  VAR __DS0FilterTable = 
    FILTER(
      KEEPFILTERS(VALUES('Products'[Brand])),
      NOT('Products'[Brand] IN {"Adventure Works"})
    )

  VAR __DS0Core = 
    SUMMARIZECOLUMNS(
      ROLLUPADDISSUBTOTAL('Products'[Brand], "IsGrandTotalRowTotal"),
      __DS0FilterTable,
      "Total_Net_Sales", 'Sales'[Total Net Sales],
      "v__of_Total_Sales", 'Sales'[% of Total Sales],
      "v__of_Total_Sales_AllSelected", 'Sales'[% of Total Sales AllSelected]
    )

  VAR __DS0PrimaryWindowed = 
    TOPN(502, __DS0Core, [IsGrandTotalRowTotal], 0, 'Products'[Brand], 1)

EVALUATE
  __DS0PrimaryWindowed

ORDER BY
  [IsGrandTotalRowTotal] DESC, 'Products'[Brand]

```

When going through the debugger though, it is not recognizing ALLSELECTED properly.
![ALLSELECTED Debuger bug](https://user-images.githubusercontent.com/45298358/191795794-1a52e2aa-8e3a-4f90-8f4f-bb515cd336df.gif)

If you need me to link to the Contoso file I have (200MB) let me know, but I think this is pretty simple to repro.

**Comment by otykier** [comment]
Thanks for reporting. We’re aware of this and currently working on a fix.

**Comment by otykier** [comment]
This is now finally fixed in [3.3.6](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.3.6). Thanks for your patience!
